MapboxNavigation/CarPlayManager.swift

```diff
@@ -645,3 +651,16 @@ extension CarPlayManager: CarPlayNavigationDelegate {
     }
 }
 #endif
+
+struct CarPlayManagerHelper {
+    
+    static var isConnectedToCarPlay: Bool {
```

Comment:
Could this property be a static computed property on CarPlayManager itself? Is the reason for splitting this struct out that would be available on all versions of iOS? If so, how about defining a stub CarPlayManager in the `#else` above, with only the `isConnectedToCarPlay` property implemented?

Reply:
> Is the reason for splitting this struct out that would be available on all versions of iOS?

Yes

> If so, how about defining a stub CarPlayManager in the #else above, with only the isConnectedToCarPlay property implemented?

👍